insights/parsers/cron_daily_rhsmd.py

```diff
@@ -0,0 +1,16 @@
+"""
+CronDailyRhsmd - file ``/etc/cron.daily/rhsmd``
+===============================================
+"""
+
+from insights.core import LogFileOutput
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.cron_daily_rhsmd)
+class CronDailyRhsmd(LogFileOutput):
```

Comment:
@huali027 I think its confusing to use `LogFileOutput` as the base parser here as there are methods/attributes in the base parser that would not work/be set.  I think you should just treat it as a text file and use the base `Parser` class.  Or if you prefer you could create a new base class for plain text parsers and provide search options to locate strings.  What do you think?

Reply:
@bfahr, sure, thanks for your review.